### PR TITLE
Aluminum and copper utility expansions, attempt 2

### PR DESCRIPTION
**Pull request by DangerNoodle**
See failed pull request: https://github.com/CleverRaven/Cataclysm-DDA/pull/18490

Aluminum ingots set to 1 volume and a density-appropriate weight, extra-light frames set to use aluminum (material and recipe). Aluminum added as options for antennas, directional antennas, radio repeater mods, and tongs. Copper wire changed to actually require copper, and made copper wire follow conservation of mass.

Changes and fixes relevant to the content of this pull request, as suggested by @mugling, have been implemented.

Aluminum foil crafting, cooking with foil, and crafting ingots will be in future pull requests. The first two are not as high on my priority list, and the last will have a recipe using native aluminum when Mining Mod is re-opened.

### CODE REVIEW

**Comment by DangerNoodle** [comment]
Relevant to the idea of making ingots: https://www.youtube.com/watch?v=lSoWxG30rb0

The maker of that video states a figure of 38-45 cans (assumed to be the 12-fluid-ounce type) to equal a pound of aluminum. With ingots being reduced to 675 grams, that would work out to an obscene 56 or 57 cans if we used the minimum of that range.

**Comment by tyrael93** [comment]
Instead of trying to mine enough aluminum to do something, which seems an incredibly daunting task just to make a pair of tongs, why not turn solar car frames into aluminum frames?
I mean, look at Teslas. A purely solar car would have to be extremely efficient in terms of weight, so the chassis MUST be made completely out of aluminum.
And recycling a car is going to be much, much easier than mining the resources from the ground!
In addition for you could just spawn more recyclers in the streets and next to garages and electronic shops. Just feed them the frames and get back your ingots.
Seems like a very clean, easy solution.

**Comment by DangerNoodle** [comment]
Perhaps. Mining and such is likely a long way away to begin with. Further ideas for aluminum sources would indeed be useful.

**Comment by Coolthulhu** [comment]
Since you're changing the xlframe material and recipe, you should also change its drops when bashed down on a vehicle.

**Comment by DangerNoodle** [comment]
Odd. Apoloies for that, I must have failed to remember that change.

File: data/json/recipes/recipe_electronics.json [hunk 1]
```diff
@@ -158,7 +158,8 @@
     [
       [ "steel_chunk", 1 ],
       [ "scrap", 3 ],
-      [ "knife_butter", 2 ]
+      [ "knife_butter", 2 ],
+      [ "material_aluminium_ingot", 1 ]
```
Comment (mugling) [inline data/json/recipes/recipe_electronics.json]: 90% of mass is lost. Also nobody will use this recipe given how available the other items are

File: data/json/recipes/recipe_electronics.json [hunk 2]
```diff
@@ -2370,7 +2374,10 @@
   "components": [
     [ [ "processor", 1 ] ],
     [ [ "scrap", 2 ] ],
-    [ [ "wire", 5 ] ],
+    [
```
Comment (mugling) [inline data/json/recipes/recipe_electronics.json]: Short length of electrical wire or 1kg of aluminium doesn't make sense

File: data/json/recipes/recipe_others.json [hunk 3]
```diff
@@ -3722,7 +3722,8 @@
      [ "steel_lump", 1 ],
      [ "steel_chunk", 4 ],
      [ "scrap", 12 ],
-     [ "pipe", 3 ]
+     [ "pipe", 3 ],
+     [ "material_aluminium_ingot", 1 ]
```
Comment (mugling) [inline data/json/recipes/recipe_others.json]: That item is still made of `steel`. We probably shouldn't mix materials like this

File: data/json/recipes/recipe_electronics.json [hunk 4]
```diff
@@ -2098,8 +2102,8 @@
   ],
   "tools": [ [ [ "boltcutters", -1 ], [ "toolset", -1 ] ] ],
   "components": [
-    [ [ "wire", 20 ], [ "wire_barbed", 20 ] ],
-    [ [ "duct_tape", 300 ] ]
+    [ [ "copper", 100 ], [ "scrap_copper", 1 ] ],
```
Comment (mugling) [inline data/json/recipes/recipe_electronics.json]: Good idea, but does it conserve mass? I'd put copper in it's own PR

**Comment by mugling** [comment]
I've merged everything except the recipes highlighted above.

Next steps:

* You could either try to work `aluminium_ingot` into more recipes, which could be challenging, or alternatively go with the suggestion for aluminium vehicles.

* Copper should be it's own PR

* Aluminium foil is up to you if you want to submit a further PR

File: data/json/recipes/recipe_electronics.json (hunk 4 above)
Comment (DangerNoodle) [inline data/json/recipes/recipe_electronics.json]: As the opening post stated at the end of the first paragraph, yes.

File: data/json/recipes/recipe_electronics.json (hunk 2 above)
Comment (DangerNoodle) [inline data/json/recipes/recipe_electronics.json]: Incorrect. Copper wire is item ID `cable`, `wire` is generic metal fence wire. One `wire` is 151 grams, 5 of them are 755 grams. I could have made the recipe 4 wire or 1 ingot (604 grams versus 675) instead.

Comment (mugling) [inline data/json/recipes/recipe_electronics.json]: I see. I'm not sure ingots and steel wire are interchangeable though?

**Comment by DangerNoodle** [comment]
As stated in comment responses, `wire` is not electrical wire, as should have been obvious from the recipe for `cable`, where an absurd 3020 grams of steel wire (plus 1200 grams of duct tape) can become 600 grams of copper wire.

File: data/json/recipes/recipe_electronics.json (hunk 2 above)
Comment (DangerNoodle) [inline data/json/recipes/recipe_electronics.json]: I had assumed that in this recipe, it is playing the part of the antenna itself.
